test.py

```diff
@@ -0,0 +1,70 @@
+# import torch.nn as nn

```

Comment:
Let’s not commit this file and trace.ts into the repo. For future notice there is a git ignored directory you can create `<repo path>/experiments` where you can put these files so they don’t show up in your changes